### PR TITLE
Change QA workflow link to internal Notion

### PR DESCRIPTION
**Pull request by Garandor**
## Description

Fix the label checker workflow to rerun when a PR is labeled and not just on push

Update QA template with lessons learned from v3.3.0 release

- Grafana
- baikal improvements
- external notifications

relates to #599 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels out of `A-` and `C-` groups to this PR
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.
- If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. You can run the `metadata_diff.yml` workflow for help. If this number is updated, then the `spec_version` must also be updated
- Verify benchmarks & weights have been updated for any modified runtime logics

### CODE REVIEW

**Comment by ghzlatarev** [comment]
While you're at this, can you also add a few things:
- [x] Link to the Telemetry page in the monitoring section.
- [x] Bullet that the person can hook up our internal testnet to this tool https://parachain-utilities.vercel.app/ to get some data like average block times, block authors, etc
- [x] Mention to notify exchange teams explicitly not just subscan

**Review by Apokalip — DISMISSED** [review]

**Comment by Garandor** [comment]
@Apokalip pls reapprove, had to fix the label checker workflow to rerun when a PR is labeled and not just on push

**Comment by Garandor** [comment]
oh shit i just forcepushed an old state and threw away all of my changes

**Review by Apokalip — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -63,15 +63,22 @@ Note: Usually update client first then runtime.
...
-### After Runtime Upgrade
-- [ ] Notify subscan team. Ensure subscan service can continue to scan calamari blocks.
+### Before Runtime Upgrade Vote
+- [ ] Prepare a governance post and submit to our forum with description and motivation for changes.
+
+### During Runtime Upgrade Vote
+- Notify all external users of `manta`, include the block number the upgrade is enacted!
+  - [ ] [Calamari Network Forum](https://forum.manta.network/c/calamari-network-governance/6)
+  - [ ] Manta/Calamari Discord Announcement
+  - [ ] Exchange Integration Teams
```
Comment (Garandor) [inline .github/ISSUE_TEMPLATE/release.md]: @Dengjianping Please add the concrete Exchanges and their contact method here ( if public )

**Comment by ghzlatarev** [comment]
@Garandor we can also decide whether to get rid of the Moonbase-Relay chain during planning. Basically all the nodes somehow dropped because of some backend AWS thing (not exactly sure what, but it wasn't a blockchain thing). And now we have to involve moonbeam people to restart us, and i'm not sure whether we should bother them.
Nvm let's do this separately.

File: .github/ISSUE_TEMPLATE/release.md (hunk 1 above)
Comment (Dengjianping) [reply .github/ISSUE_TEMPLATE/release.md]: I have added a notion page.
https://www.notion.so/mantanetwork/Exchanges-3rd-Infrastructures-b089e136a14b430ea405400311b362cb

**Review by ghzlatarev — APPROVED** [review]

**Review by Dengjianping — APPROVED** [review]

**Pull request merged**